apps/core/src/migration/util/convertSiteWiseMonitorToApplicationDefinition.ts

```diff
@@ -0,0 +1,11 @@
+import { DashboardDefinition } from 'src/dashboards/entities/dashboard-definition.entity';
+
+export const convertSiteWiseMonitorToApplicationDefinition = (
+  sitewiseMonitorDashboardDefinition?: string,
+): DashboardDefinition => {
+  // TODO: add actual definition conversion
```

Comment:
assuming this will be a followup PR?

Reply:
Yes

Reply:
This part will be the most tricky part of the migration so will take some time, so I wanted to get in the rest first.